README.md

```diff
@@ -1 +1,148 @@
-# layout
\ No newline at end of file
+# Wolox Layout
+
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-layout.svg)](https://circleci.com/gh/Wolox/wolox-layout)
+[![npm](https://img.shields.io/npm/v/wolox-layout.svg)](https://www.npmjs.com/package/wolox-layout)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+
+## Features
+
+* `separations.scss` file contains all the margin and padding class generators for your own stylesheet.
+* `layout.scss` contains a series of classes that can be used to define a page's layout using flexbox.
+
+## Installation
+
+1.  intall the package with `npm` or `yarn`
+
+#### npm
+
+```bash
+npm i wolox-layout
+```
+
+#### yarn
+
+```bash
+yarn add wolox-layout
+```
+
+2- Include the wolox-bootstrap.scss before the index file of your CSS/SCSS to have paddings, margins, and layout already generated all toghether. Or you can import them separately.
+
+If you want to import the 'wolox-bootstrap' in this way:
+
+```scss
+@import '~wolox-bootstrap/wolox-bootstrap';
+ // or
+@import '~wolox-bootstrap/layout';
+@import '~wolox-bootstrap/separations';
+```
+
+## Usage
+
+### General import of Wolox-boostrap
+
+This will generate a series of classes like:
+```sass
+.m-0 { margin: 0; }
+.m-1 { margin: 10px; }
+...
+.m-20 { margin: 200px; }
+
+.m-top-0 { margin: 0; }
+.m-top-1 { margin: 10px; }
+...
+.m-top-20 { margin: 200px; }
+
+@media (max-width: 960px) {
```

Comment:
However we allow the user to generate them

Reply:
we shouldn't let the users generate paddings, paddings should be declared in it's component classes since they are part of the component styles. only margins should be added separately